examples/meta_example.py

```diff
@@ -0,0 +1,36 @@
+# # Meta-analysis: an example of use of the KDA & MKDA method.
+
+# +
+import pandas as pd
+from nilearn import plotting
+
+from neuroquery import datasets
+from neuroquery.img_utils import coordinates_to_arrays
+from neuroquery.meta_analysis import KDA, MKDA
+
+# +
+# Load the coordinates from the NeuroQuery dataset.
+coordinates = pd.read_csv(datasets.fetch_peak_coordinates())
```

Comment:
It would be great if the example could select studies that contain a certain term using the tfidf features provided with the dataset. in fact, I think we will probably want a function for that

Reply:
also that will speed up the example wrt to building maps for the whole dataset

Reply:
Yes, is `img_utils.py` a good place for this function?

Reply:
I'm currently implementing this function. 
The problem is that, as it stands, the user should give the tfidf and vocabulary files to be able to select the studies related to a given keyword.
I see 2 options depending on whether you want the users to be able to specify their own data (i.e coordinates + tfidf + vocabulary files).

- **Option 1: users can specify their data.** In this case, we should send to the users the coordinates + tfidf + vocabulary file as a block inside `fetch_peak_coordinates`. Currently `fetch_peak_coordinates` sends the coordinates alone but I'm not sure that coordinates alone make sens without tfidf and vocabulary files. Then in the function selecting the pmids related to a keyword, we would ask to the users their coordinates, tfidf and vocabulary data. This way should let them free to give their own data.

 - **Option 2: users cannot specify their data.** In this case, no need to ask the users to give the tfidf and vocabulary files. They just give the coordinates data frame and a keyword and the function look inside the neuroquery data folder for the tfidf and vocabulary files.

What do you think?

Reply:
I think Option 1, it's best if users can provide their own data. we can provide defaults so that we use the data provided by neuroquery if they don't specify anything

Reply:
most users will use the default. otherwise as you say they need to provide tfidf + vocabulary + pmids (or possibly a sparse dataframe?), and the coordinates dataframe

Reply:
Great, I'm pushing an implementation of it.